FTB/Signatures/tests/test_CrashInfo.py

```diff
@@ -1032,11 +1029,11 @@ def test_GDBParserTestCrashAddressSimple():
     # Note: The crash address here can also be 0xf7600000 because the double quadword
     # move can fail on the second 8 bytes if the source address is not 16-byte aligned
     assert (GDBCrashInfo.calculateCrashAddress("movdqu 0x40(%ecx),%xmm4", registerMap32) ==
-            int32(uint32(0xf75ffff8)))
```

Comment:
Can you keep the original value here using `int32`/`uint32` as you defined it in `CrashInfo.py`? Turning this into a decimal makes the test harder to understand.